c-bindings/handles.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+)
+
+const maxID = 1000
```

Comment:
I think we should raise this limit to at least `100 000` if we're using it for JSON values.